eng/code-quality-reports/src/main/java/com/azure/tools/checkstyle/checks/DisallowedWordsCheck.java

```diff
@@ -0,0 +1,152 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.tools.checkstyle.checks;
+
+import com.puppycrawl.tools.checkstyle.api.AbstractCheck;
+import com.puppycrawl.tools.checkstyle.api.DetailAST;
+import com.puppycrawl.tools.checkstyle.api.TokenTypes;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+public class DisallowedWordsCheck extends AbstractCheck {
+    private final String[] DEFAULT_VALUES = new String[] { "URL", "HTTP" };
```

Comment:
Perhaps remove this field and have it just configured through the XML config?

Reply:
Well, really, it could be taken away. The only thing that could happen is that someone using the Checkstyle would need to look up how to use it and really that should be a given.

Reply:
Ended up taking the property away since someone should look at the class if it doesn't work as they think it should. A default didn't really help or make sense.